src/liboslexec/oslexec_pvt.h

```diff
@@ -594,7 +599,8 @@ class ShaderInstance {
 
     /// Does it appear that the layer is completely unused?
     ///
-    bool unused () const { return run_lazily() && ! outgoing_connections(); }
+    bool unused () const { return run_lazily() && ! outgoing_connections()
```

Comment:
Doesn't run_lazily() include the other flags? Should this be just !outgoing_connections() && !render_outputs()?